README.md

```diff
@@ -1,5 +1,81 @@
 # PayPal iOS SDK
 
-![GitHub Actions Tests](https://github.com/paypal/iOS-SDK/workflows/Tests/badge.svg)
+Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payments in your iOS app.
 
-[WIP] One merchant integration point for all of PayPal's services
+## Support
+
+The PayPal iOS SDK supports a minimum deployment target of iOS 13+ and requires Xcode 13+. See our [Client Deprecation policy](https://developer.paypal.com/braintree/docs/guides/client-sdk/deprecation-policy/ios/v5) to plan for updates.
+
+### Package Managers
+This SDK supports:
+
+* CocoaPods
+* Swift Package Manager
+
+### Languages
+
+This SDK supports Swift 5.1+. This SDK is written in Swift.
```

Comment:
Should we also call out our minimum OS target here?

Reply:
That seems redundant to me since it's stated a few lines above - but can add it again if we think it's helpful.

Reply:
Whoops, missed it above. No need!